src/SIL.LCModel/Application/ApplicationServices/XmlImportData.cs

```diff
@@ -605,7 +603,7 @@ private void CopyCustomFieldData(ICmObject cmoOld, ICmObject cmoNew)
 					case CellarPropertyType.Boolean:
 						break;
 					case CellarPropertyType.Integer:
-						int val = sda.get_IntProp(cmoOld.Hvo, flid);
+						int val = sda!.get_IntProp(cmoOld.Hvo, flid);
```

Comment:
now that you're throwing when sda is null I don't think you need `!` here or any of the other lines below

Reply:
Done